website/docs/getting-started/setup/installation-guides/docker/README.md

```diff
@@ -5,172 +5,157 @@ sidebar_position: 1
+If the image doesn't exist locally, this command downloads the Docker images and starts its services. You can follow the logs with the following command:
 
 ```bash
 docker logs -f appsmith
 ```
 
-You should see a message `Appsmith is Running!` once the container is ready. The message is also logged and available in server logs(<mark>`stacks/logs/backend/backend.log`</mark>).
+Once the container is ready, you should see `Appsmith is Running!` in the Docker logs. This message is also logged and available in server logs in (<mark>`stacks/logs/backend/backend.log`</mark>).
 
 ![Appsmith is running message](/img/InstallationGuides__Docker__AppsmithRunningMessage.png)
 
-:::tip
-If you are **unable** to **locate** the message but can **access** Appsmith via **localhost** or a **custom domain** you have **created**, then Appsmith is up and running.
-:::
-
-If this is your first time using docker, you should expect a welcome page similar to the one below.
+Congratulations, your Appsmith server should be up and running now. You can access it at [http://localhost](http://localhost). If this is your first time setting up using Docker, you should expect a welcome page similar to the one below.
 
 ![Welcome Page](</img/image_(1)_(1).png>)
 
-:::tip
-Congratulations! Your Appsmith server should be up and running now. You can access it at [http://localhost](http://localhost).
-:::
-
-### Updating Appsmith (with docker-compose)
+### Updating Appsmith manually
 
-To update Appsmith (configured with docker-compose) manually, go to the root directory of your setup and run the following command:
+To update Appsmith (configured with docker-compose) manually, go to the root directory of your installation and run the following command:
 
 ```
 docker-compose pull && docker-compose rm -fsv appsmith && docker-compose up -d
 ```
 
-### Enabling Appsmith Auto-updates (with docker-compose)
+### Enabling Appsmith auto-updates
 
-If your Appsmith setup does not have auto-update enabled (i.e. it will not have the Watchtower container running along with Appsmith in the host machine). You can enable auto-update by following the following steps:
+Automatic updates are disabled by default in the docker-compose.yml file. If you would like to enable automatic updates for Appsmith, please follow these steps:
 
-1. Go to the root directory of your Appsmith set up and run:
+1. Go to the root directory of your Appsmith instance and run:
 
 ```
 docker-compose down
 ```
 
-1. Open the `docker-compose.yml` file with any text editor and uncomment all the lines that are commented out (lines 13-23).
-2. Run the command:
+2. Open the `docker-compose.yml` file with any text editor and uncomment all the lines that are commented out (lines 13-23).
+3. Save the file and run:
 
 ```
 docker-compose up -d
 ```
 
-## Explore Appsmith (without docker-compose)
+:::info
```

Comment:
This piece of info seems misplaced

Reply:
This info about auto-backups is placed here because they are directly related to the auto-updates feature; before any auto-update, Appsmith makes a backup. (And scheduled backups aren't implemented yet, auto-backup only works as a part of auto-update)

Users may be hesitant to enable auto-updates in case their apps will get broken by an update that they didn't have control over. But if they see this info and read about the auto-backups, they can be sure that there is a failsafe in the case of breaking changes.

Does that make sense? I like the `:::info` there, but I'm open to other ideas if it should be somewhere else.